pom.xml

```diff
@@ -0,0 +1,20 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.ehcache.sample</groupId>
+  <artifactId>parent</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
you're not this parent in the projects (other than fullstack since fullstack is bound to use spring boot parent)
and this is also a reactor pom including modules as such; and I'm not sure we want that.
all in all, I don't think we should use such a parent pom ; those projects are self services projects, you checkout and run only 1.
I understand it would have been nice to have a "centralized" ehcache version, but I believe those modules are better off "unbound" to anything